packages/baseclient/config/.gitignore

```diff
@@ -0,0 +1 @@
+projectconfig.js
```

Comment:
is this intended to have a gitignore in the config folder?

Reply:
I've removed this in 82d0daf since `projectconfig` is not required anymore